components/public-api/gitpod/experimental/v2/configuration.proto

```diff
@@ -0,0 +1,88 @@
+syntax = "proto3";
+
+package gitpod.experimental.v2;
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v2";
+
+import "google/protobuf/timestamp.proto";
+import "gitpod/experimental/v2/pagination.proto";
+
+
+message Configuration {
+    string id = 1;
+    string organization_id = 2;
+    string name = 3;
+    string clone_url = 4;
+    google.protobuf.Timestamp creation_time = 5;
+    PrebuildSettings prebuild_settings = 6;
+    WorkspaceSettings workspace_settings = 7;
+}
+
+message PrebuildSettings {
+    optional bool enabled = 1;
+    optional string branch_matching_pattern = 2;
+    optional BranchMatchingStrategy branch_strategy = 3;
+    optional int32 prebuild_interval = 4;
+    optional string workspace_class = 5;
+}
+
+enum BranchMatchingStrategy {
+    BRANCH_MATCHING_STRATEGY_UNSPECIFIED = 0;
+    BRANCH_MATCHING_STRATEGY_DEFAULT_BRANCH = 1;
+    BRANCH_MATCHING_STRATEGY_ALL_BRANCHES = 2;
+    BRANCH_MATCHING_STRATEGY_MATCHED_BRANCHES = 3;
+}
+
+message WorkspaceSettings {
+    optional string workspace_class = 1;
+}
+
+service ConfigurationService {
```

Comment:
Is it old project service? Does it mean that we are going eventually remain projects to configurations on UI and in docs?

I am worry that we will introduce problems if we use different language for APIs.

Reply:
Yes, we are renaming projects to (repository) configurations. cc @loujaybee